packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/modelparts/ModelMetadata.java

```diff
@@ -44,9 +44,12 @@ public class ModelMetadata extends SupportAdditionalProperties implements Serial
 	@TSOptional
 	private Map<String, Object> timeseries;
 
-	/* Link user input string `source` to a parameter/variables of a model. */
-	@TSOptional 
-	private Map<String, Object> sources;
+
+	@TSOptional
+	private Map<String, Object> initials;
+
+	@TSOptional
+	private Map<String, Object> parameters;
```

Comment:
I could see the metadata becoming a little bloated, so Ive added initials and parameters here where we can place any metadata we want: i.e

```json
...metadata.initials: {
    s: {
         description: '',
         unit: '',
         source: '',
    }
}
```
TODO move timeseries over as well

Reply:
Can you elaborate on what `metadata.initials` and `metadata.parameters` are used for? 

If you are talking about metadata for model variables they have their own schemes in their respective model-representation, and if kept out side of that it may cause problems downstream.

Reply:
Forgive me if Im wrong, I thought the AMR schema was pretty static aside from `metadata` key where we could add anything we wanted, which is why we used the timeseries in metadata.  So for instance an initial has this structure:
```
export interface Initial {
    target: string;
    expression: string;
    expression_mathml: string;
}
```

But now I want an initial to have a `unit`, `description`, `source`.  I would happily add it to the Initial type, but as I mentioned before I recall we couldn't touch these things.

So now similar to the timeseries, I have a map on the metadata for initials which will have these additional metadata parameters, which we can then map in the front end.

We can discuss in the meeting further!